src/mscorlib/corefx/System/Globalization/JapaneseCalendar.Win32.cs

```diff
@@ -7,12 +7,51 @@ namespace System.Globalization
 {
     public partial class JapaneseCalendar : Calendar
     {
-        public static int GetJapaneseEraCount()
+        private static EraInfo[] GetJapaneseEras()
         {
-            return WinRTInterop.Callbacks.GetJapaneseEraCount();
+            int erasCount = WinRTInterop.Callbacks.GetJapaneseEraCount();
```

Comment:
We cannot assume we are running on Windows platform support WinRT. we have to depend on the Win32 APIs here I think